packages/formation/scripts/build.js

```diff
@@ -1,13 +1,12 @@
 /* eslint-disable no-console */
-const mkdirp = require('mkdirp');
-const fs = require('fs-extra');
 const webpack = require('webpack');
 const webpackConfig = require('../../../webpack.config.prod.js');
+const rimraf = require('rimraf');
 
 console.log('Starting Formation build');
 console.log('Cleaning old build');
-mkdirp.sync('./dist/img');
```

Comment:
Webpack can do this. We could potentially ditch this build script and use webpack's CLI.